addons/godot-xr-tools/assets/PlayerBody.gd

```diff
@@ -147,6 +157,13 @@ func set_enabled(new_value):
 func get_enabled():
 	return enabled
 
+func set_player_radius(new_value: float) -> void:
+	player_radius = new_value
+	_collision_node.shape.radius = new_value
```

Comment:
ok be careful here, the first time this gets called when a script loads, `_collision_node` won't be initialised yet. You'll see in much of the code I use a pattern like this:
```
export var some_value setget set_some_value

func set_some_value(new_value):
    some_value = new_value
    if inside_tree():
      _update_some_value()

func _update_some_value():
    # do stuff here that needs doing

func _ready():
    # now that we're ready, we can propogate all our defaults
    _update_some_value()

Reply:
I've applied this pattern to all the appropriate setters.